### PR TITLE
Lection 3 homework

### PR DESCRIPTION
**Pull request by dispersik**

### CODE REVIEW

File: homework3.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+func average(arr [6]int) float64 {
+	sum := 0
+	for _, element := range arr {
+		sum += element
+	}
+	return float64(sum) / 6
+}
+
+func max(s []string) string {
+	// function that returns the longest word from
```
Comment (ikamornikau) [inline homework3.go]: Please see https://blog.golang.org/godoc and make appropriate changes.

File: homework3.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	// the slice of strings (the first if there are more
+	// than one).
+	// Input -> ("one", "two", "three")
+	// Output -> ("three")
+	// Input -> ("one", "two")
+	// Output -> ("one")
+	k := 0
```
Comment (ikamornikau) [inline homework3.go]: What is `k`? Please rename.

**Review by ikamornikau — CHANGES_REQUESTED** [review]
Please address the requested changes.

**Comment by ikamornikau** [comment]
BTW where is `printSorted()`?

File: homework3.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	fmt.Println("Out: ", average(a), "\n")
+
+	fmt.Println("max() func tests\n")
+	str := []string{"one", "zero", "five", "three"}
+	fmt.Printf("In: %s\n", str)
+	fmt.Printf("Out: %s\n\n", max(str))
+
+	fmt.Println("reverse() func tests\n")
+	arr := []int64{0,1,2,3,4}
+	fmt.Println("In: ", arr, "len = ", len(arr), "cap = ", cap(arr))
```
Comment (VolhaVetrava) [inline homework3.go]: `fmt.Printf` would look better here

**Review by VolhaVetrava — CHANGES_REQUESTED** [review]

**Pull request closed without merging**